libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -21,8 +21,9 @@ It has the same interface and semantics as [`cuda::std::barrier`], with the
 
 ## Barrier Operations
 
-| [`cuda::barrier::init`]                 | Initialize a `cuda::barrier`. `(friend function)`                 |
-| [`cuda::device::barrier_native_handle`] | Get the native handle to a `cuda::barrier`. `(function template)` |
+| [`cuda::barrier::init`]                 | Initialize a `cuda::barrier`. `(friend function)`                       |
+| [`cuda::device::barrier_native_handle`] | Get the native handle to a `cuda::barrier`. `(function template)`       |
```

Comment:
...I don't know why this says that it's a template, but it is not and has, to my knowledge, never been a template... (I recognize that that's in the repo already, but would be nice to clean this up.)

Reply:
changed to function.